src/stream.js

```diff
@@ -0,0 +1,6 @@
+export default async () => {
+  // eslint-disable-next-line no-undef
+  const stream = await globalThis.process?.release?.name ?
+    import('stream') : import('readable-stream')
```

Comment:
Do you really need this, webpack don't do this job already ?

Reply:
hum actually i have no idea, i saw browserify use another package, and i don't assume that this client is webpack-dependant :thinking: